### ISSUE TITLE
Can't launch the game 

### ISSUE THREAD

**Issue by DucViet0202** [FireMuffin303/What-it-should-be#2]
here is the log
[latest.log](https://github.com/user-attachments/files/18140285/latest.log)

**Comment by FireMuffin303** [comment]
Hi. It seems What It Should Be's Sugar Cane modification is conflicted to Survivality's Sugar Cane modification.

```
[18:43:54] [main/WARN]: @ModifyConstant conflict. Skipping wisb.mixins.json:bonemealable.SugarCaneBlockMixin from mod wisb->@ModifyConstant::wisb$randomTick(I)I with priority 1000, already redirected by survivality.mixins.json:block.SugarCaneBlockMixin from mod survivality->@ModifyConstant::survivality_increaseCactusHeight(I)I with priority 1000
```

This bug will be fix in the 1.0.2 patch. Right now I suggest you to uninstall What It Should Be since it still in Beta state.

Thank you for report!

**Comment by DucViet0202** [comment]
Okay, thanks
